### DISCUSSION TITLE
Add function to disable/enable "Edit in Notepad" context menu item

### DISCUSSION THREAD

**Discussion by soredake** [farag2/Sophia-Script-for-Windows#597] [Ideas]
https://www.elevenforum.com/t/add-or-remove-edit-in-notepad-context-menu-in-windows-11.20485/

**Comment by farag2** [comment]
Do not want to add such stuff. I'm awared.

**Reply by soredake** [reply]
Yet, there are functions for "Edit with Clipchamp" and "Edit with Photos"

**Reply by farag2** [reply]
But I don't like them and allow to remove)
